contributing/fetch.sh

```diff
@@ -0,0 +1,29 @@
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-beneficiary-2014.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-carrier-1999-1999.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-carrier-1999-2000.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-carrier-1999-2001.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-carrier-2000-2000.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-carrier-2000-2001.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-carrier-2000-2002.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-carrier-2014-2014.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-carrier-2014-2015.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-carrier-2014-2016.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-inpatient-1999-1999.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-inpatient-1999-2000.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-inpatient-1999-2001.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-inpatient-2000-2000.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-inpatient-2000-2001.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-inpatient-2000-2002.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-inpatient-2014-2014.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-inpatient-2014-2015.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-inpatient-2014-2016.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-pde-2014.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-pde-2015.rif \
+    https://bfd-public-test-data.s3.amazonaws.com/data-synthetic/2020-1-25-mbi-with-negative-ids/synthetic-pde-2016.rif
+
```

Comment:
The most current folder is 2020-02-27-add-outpatient. Could we make this script detect the most current version?

Reply:
I don't know enough about s3, but I'll look into it.

Have you found a better method to load data locally? I'd like to get the use of git patches reduced as much as possible.

Reply:
Captured in: https://jira.cms.gov/browse/BLUEBUTTON-1993